packages/sdk/src/libs/coingecko/coingecko.test.ts

```diff
@@ -2,7 +2,7 @@ import assert from "assert";
 import Coingecko from ".";
 
 // this requires e2e testing, should only test manually for now
-describe.skip("coingecko", function () {
+describe("coingecko", function () {
```

Comment:
How are we walling off manual/e2e tests from the other tests in ci?

Side note: are the compiled test files currently being exported to npm?

Reply:
hm that should be skipped actually. i have meant to skip all non unit tests.  i dont have a great solution for how to deal wiht e2e style tests. 
tests are not compiled with the production build afaik!